package.json

```diff
@@ -51,13 +51,15 @@
   },
   "homepage": "https://github.com/ERC725Alliance/erc725.js#readme",
   "devDependencies": {
+    "@types/chai": "^4.3.1",
     "@types/isomorphic-fetch": "^0.0.36",
     "@types/mocha": "^9.1.1",
     "@types/node": "^15.12.2",
     "@types/sinon": "^10.0.11",
     "@typescript-eslint/eslint-plugin": "^5.25.0",
     "@typescript-eslint/parser": "^5.25.0",
     "assert": "^2.0.0",
+    "chai": "^4.3.6",
```

Comment:
Do we have chai and mocha now? Why both?

Reply:
`chai` is used for assertions - it can be used along with Mocha - it doesn't replace it.

It allows us to write, imo, more explicit/easier to understand assertions:

![image](https://user-images.githubusercontent.com/477945/172367956-ef8ff56a-df43-4ca7-bae2-a0178bafc169.png)

Reply:
Nice. Thanks for clarification :)